packages/shared/src/styles/components/buttons.css

```diff
@@ -274,6 +274,10 @@
   @apply bg-surface-hover hover:bg-surface-active;
   @apply text-text-primary;
 
+  &.simplified {
+    @apply bg-[unset];
+  }
```

Comment:
Why do we need this?

Reply:
There is a default bg for the default, on this simplified version, we should not have bg.